src/pages/components/anchor.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Anchor
+description: >
+  The Anchor component is a link used to navigate from one location to another.
+package: '@zendeskgarden/react-buttons'
+propSheets:
+  - buttons/src/elements/Anchor.tsx
+---
+
+import AnchorDefault from '../../examples/anchor/AnchorDefault.tsx';
+import AnchorDefaultCode from '!!raw-loader!../../examples/anchor/AnchorDefault.tsx';
+import AnchorExternal from '../../examples/anchor/AnchorExternal.tsx';
+import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExternal.tsx';
+import AnchorDanger from '../../examples/anchor/AnchorDanger.tsx';
+import AnchorDangerCode from '!!raw-loader!../../examples/anchor/AnchorDanger.tsx';
```

Comment:
Could we include these imports next to their respective examples? With the `raw-loader` usage this helps with co-location.

Reply:
I will of course defer, but are we open to this approach in any way? Eventually (probably sooner than we think) we will have contributors on these pages that aren't Devs and so every effort we can make to have the Markdown/content feel like Markdown without import statement or other parts to ignore would be amazing from a contributor standpoint.

Reply:
My preference is for co-location too since it helps me keep track of the example and their respective imports. If I need to make adjustments to either an import or its example, I would have to use mental energy to recall where each is in the file. It would require lower cognition to have them colocated. 

I think contributors need to be comfortable with `.mdx` files to contribute code changes. In these cases, they should be comfortable copying/pasting/moving the import statements. If they are only updating copy text, then it shouldn't be a concern. I think in the future when we define contributors (only those with Abstract tokens?) we could make adjustments.

Reply:
I think I'm solving for problems we don't have again. I'm sure over time we'll uncover optimizations that simplify this syntax anyway. Will amend